gmm_client/src/client.erl

```diff
@@ -0,0 +1,105 @@
+%%%-------------------------------------------------------------------
+%%% @author Svatopluk
+%%% @copyright (C) 2021, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 10. maj 2021 16:54
+%%%-------------------------------------------------------------------
+-module(client).
+-author("Svatopluk").
+
+
+%% API
+-export([get_id_list/0, get_id_number/0, get_user_info/1, delete_user/1, add_user/1, test/0]).
+
+test()->
+  ok.
+
+url_address() ->
+  "localhost:8080/".
+
+% 
+
+get_request(URL)->
+  Method = get,
+  Headers = [],
+  Payload = <<>>,
+  Options = [],
+  {ok, StatusCode, RepsHeaders, ClientRef} = hackney:request(Method, URL, Headers, Payload, Options),
+  {ok, StatusCode, RepsHeaders, ClientRef}.
+
+% reads body of get request
+read_body(ClientRef) ->
+  {ok, _} = hackney:body(ClientRef).
```

Comment:
Jeżeli już robisz pattern matching do `{ok, Body}`, to chyba warto po prostu już zwrócić to body. Informacja o błędzie i tak nigdy nie wyjdzie z tej funkcji, tylko spowoduje błąd